templates/includes/_header.html

```diff
@@ -28,6 +28,13 @@
             <a href="{% url 'releases-most-recent' %}" class="hover:no-underline font-semibold dark:font-medium {% if request.path|slice:"0:10" == '/releases/' %}dark:text-orange text-orange dark:hover:text-white hover:text-slate {% else %}dark:text-white text-slate dark:hover:text-orange hover:text-orange {% endif %} link-none visited:text-black">Releases</a>
           </nav>
           <nav class="flex items-center space-x-3 lg:space-x-5 h-[30px]" x-data="{ 'searchOpen': false }">
+            <span class="relative">
+              <i id="gecko-search-button" data-current-boost-version="1_83_0" data-theme-mode="light" data-font-family="Cairo, sans-serif" class="inline -mt-1 cursor-pointer text-slate fas fa-search dark:text-white/50 dark:hover:text-orange hover:text-orange"></i>
```

Comment:
@ashtum what are the data attributes doing? Is that something boost-gecko is using?

Reply:
Yes, that's a way to pass some parameters.
There is a MutationObserver in boost-gecko to react to the changes of these attributes as well.(like toggling dark/light modes).